python/lsst/ts/salobj/base.py

```diff
@@ -130,6 +138,59 @@ class ExpectedError(Exception):
     pass
 
 
+def angle_diff(angle1, angle2):
+    """Return angle1 - angle2 wrapped into the range [-180, 180) deg.
+
+    Parameters
+    ----------
+    angle1 : `astropy.coordinates.Angle` or `float`
+        Angle 1; if a float then in degrees
+    angle2 : `astropy.coordinates.Angle` or `float`
+        Angle 2; if a float then in degrees
+
+    Returns
+    -------
+    diff : `astropy.coordinates.Angle`
+        angle1 - angle2 wrapped into the range -180 <= diff < 180 deg.
+    """
+    return angle_wrap_center(
+        astropy.coordinates.Angle(angle1, u.deg)
+        - astropy.coordinates.Angle(angle2, u.deg)
+    )
+
+
+def angle_wrap_center(angle):
+    """Return an angle wrapped into the range [-180, 180] deg.
+
+    Parameters
+    ----------
+    angle : `astropy.coordinates.Angle` or `float`
+        Angle; if a float then in degrees
+
+    Returns
+    -------
+    wrapped : `astropy.coordinates.Angle`
+        angle wrapped into the range -180 <= wrapped < 180 deg.
```

Comment:
In line 163 you define the range as [-180, 180] while here you define it as 180 <= wrapped < 180.